### ISSUE TITLE
undefined method `task' for Sinatra::Application:Class

### ISSUE THREAD

**Issue by LeviRosol** [resque/resque#934]
I don't know where to put this, but because the error seems to have stemmed from resque, I'm putting it here.

**Background / Problem**
For a PoC I'm working on where I need a sinatra app using resque I started with this demo app: https://github.com/arol/sinatra-resque-mongodb-clockwork-bootsrap

After adding sinatra-contrib to my gemfile, creating a new gemset, bundle installing and then starting the app I kept getting the following error:

```
undefined method `task' for Sinatra::Application:Class
```

It stemmed from my require 'resque/tasks' line within the Rakefile. 

**Solution**
After a lot of trial and error and failed google searches I ultimately found that the latest version of sinatra-contrib was causing the error. Simply set the version of sinatra-contrib to 1.3.1 in your gemfile, delete your lock file, bundle install, foreman start, and the error will go away.

Does this dig into the actual issue? No. Does it help future people with a Google search for the error? Yup.

**TL;DR;**
If you are getting this error with your sinatra app that includes sinatra-contrib and resque

```
undefined method `task' for Sinatra::Application:Class
```

set your sinatra-contrib version to 1.3.1

**Comment by mariogmz** [comment]
@masonforest you're a hero!
